4.2.0/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM jetty:9-jre8
+
+ENV DATA_DIR /catalogue-data
+ENV JAVA_OPTS -Dorg.eclipse.jetty.annotations.AnnotationParser.LEVEL=OFF \
+        -Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true \
+        -Xms512M -Xss512M -Xmx2G -XX:+UseConcMarkSweepGC \
+        -Dgeonetwork.resources.dir=${DATA_DIR}/resources \
+        -Dgeonetwork.data.dir=${DATA_DIR} \
+        -Dgeonetwork.codeList.dir=/var/lib/jetty/webapps/geonetwork/WEB-INF/data/config/codelist \
+        -Dgeonetwork.schema.dir=/var/lib/jetty/webapps/geonetwork/WEB-INF/data/config/schema_plugins 
+
+USER root
+RUN apt-get -y update && \
+    apt-get -y install curl && \
+    rm -rf /var/lib/apt/lists/* && \
+    mkdir -p /${DATA_DIR} && \
+    chown -R jetty:jetty ${DATA_DIR} && \
+    mkdir -p /var/lib/jetty/webapps/geonetwork && \
+    chown -R jetty:jetty /var/lib/jetty/webapps/geonetwork
+
+USER jetty
+ENV GN_FILE geonetwork.war
+ENV GN_VERSION 4.2.0
+ENV GN_DOWNLOAD_MD5 3351b899162b9260459c6c76d1d8f446
```

Comment:
I'm getting this error when trying to build the image from `4.2.0`:

```
md5sum: WARNING: 1 computed checksum did NOT match.
```

MD5 defined in the Dockerfile is not valid. ` GN_DOWNLOAD_MD5` is set to `3351b899162b9260459c6c76d1d8f446` but it should be `5300f81f24f1585d969e667d922e8e8e`.

Reply:
Updated.